docs/release/releases/release-1.10.md

```diff
@@ -0,0 +1,36 @@
+# Cluster API v1.10
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.10` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 16th December 2024   | week 1   |
```

Comment:
What about moving the start of the release cycle to Jan 6h, I don't think that nothing meaningful will happen during xMas break

Note: This will will shorten the release cycle from 19 to 17 weeks to keep us on track for 3 release also in 2025. Also, we already had a few 17 weeks release cycles, so I think it is ok.

Reply:
No strong opinion, but may be good to do for the folks that will be part of the release team :-)